tests/test_functions.py

```diff
@@ -455,3 +456,42 @@ def test_class_in_main_is_in_right_module():
     resp = client.get("/")
 
     assert resp.status_code == 200
+
+
+def test_function_returns_none():
+    source = TEST_FUNCTIONS_DIR / "returns_none" / "main.py"
+    target = "function"
+
+    client = create_app(target, source).test_client()
+    resp = client.get("/")
+
+    assert resp.status_code == 500
+
+
+def test_legacy_function_check_env():
+    source = TEST_FUNCTIONS_DIR / "http_check_env" / "main.py"
+    target = "function"
+
+    os.environ["ENTRY_POINT"] = target
```

Comment:
This has the effect of setting the environment variable across the entire test suite run, which might cause unintentional behavior in tests that follow this. We should use [`monkeypatch.setenv`](https://docs.pytest.org/en/3.0.1/monkeypatch.html#_pytest.monkeypatch.MonkeyPatch.setenv) instead.